broker/cloud_functions/check_cue_response/README.md

```diff
@@ -0,0 +1,24 @@
+# Check cue response
+
+This Cloud Function checks whether the broker responded appropriately to the auto-scheduler's cue. If it has not, "Critical" errors are raised which trigger a GCP alerting policy.
```

Comment:
Is there one Critical error raised or multiple?

Reply:
One critical error per component checked (vm, dataflow job). I'll clarify in the text.